auth/server/src/main/java/com/seagates3/model/AuthIEMAlert.java

```diff
@@ -1,6 +1,73 @@
 package com.seagates3.model;
 
+import com.seagates3.s3service.IEMRestClient;
+import com.seagates3.s3service.S3HttpResponse;
+
+import com.amazonaws.util.json.JSONException;
+import com.amazonaws.util.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
 public
 class AuthIEMAlert {
-  // TODO Base template class for IEM alerts
+
+ private
+ final static Logger LOGGER =
+     LoggerFactory.getLogger(AuthIEMAlert.class.getName());
+ public
+  static final String LDAP_EX = "0040010001";
```

Comment:
Why we repeating these code definitions here?
Cant we use the same from IEMUtil.java

Reply:
had a discussion with Basav earlier and we decided to keep this separate

Reply:
@bkirunge7

Reply:
ok approving.